internal/handler/msg_updateuser.go

```diff
@@ -96,11 +96,51 @@ func (h *Handler) MsgUpdateUser(ctx context.Context, msg *wire.OpMsg) (*wire.OpM
 		return nil, lazyerrors.Error(err)
 	}
 
+	if mechanisms == nil {
+		return nil, handlererrors.NewCommandErrorMsg(
+			handlererrors.ErrBadValue,
+			"mechanisms field must not be empty",
+		)
```

Comment:
This condition is not tested by integration test and it doesn't seem correct. I feel it's possible to change password without specifying mechanisms, don't you think?

Reply:
Yes that's correct I will remove this